CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,129 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
```

Comment:
Minor thing, but we have used `-` character elsewhere for bullet points. To be systematic, these could still be changed to follow the same convention.

Reply:
I followed the GitHub template/convention for file location since I figured that way it would be in the standard location - but we can  move it to Docs pretty easily. Same thing with the bullet point convention though that is easily changable.

Reply:
Yep, by default GH uses root, but docs is also allowed and better here because we want the file to be visible on our docs.